src/Raven.Server/Documents/Handlers/Admin/AdminConfigurationHandler.cs

```diff
@@ -112,6 +121,21 @@ public async Task PutSettings()
             NoContentStatus(HttpStatusCode.Created);
         }
 
+        private List<string> GetUpdatedSettingsKeys(Dictionary<string, string> updatedSettings)
+        {
+            var keys = new List<string>();
+
+            foreach (var kvp in updatedSettings)
+            {
+                var currentValue= Database.Configuration.GetSetting(kvp.Key);
```

Comment:
this is different than in 6.0, why?
Also this takes into account settings from ENV and File IIRC

Reply:
We should just compare DatabaseRecord.Settings?

Reply:
Changed